etcdctl/command/util.go

```diff
@@ -221,6 +221,10 @@ func mustNewClient(c *cli.Context) client.Client {
 		err := hc.Sync(ctx)
 		cancel()
 		if err != nil {
+			if err == client.ErrNoEndpoints {
+				fmt.Fprintf(os.Stderr, "Failed to get any client endpoints when synchronizing through %s.\n", strings.Join(eps, ","))
```

Comment:
"Failed to sync cluster with endpoints [%v]". Ideally, we can print out the errors.